automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookInteractionSpec.scala

```diff
@@ -148,31 +149,35 @@ class NotebookInteractionSpec extends FreeSpec with LeonardoTestUtils with Befor
       Orchestration.billing.addUserToBillingProject(billingProject.value, ronEmail, Orchestration.billing.BillingProjectRole.User)(hermioneAuthToken)
 
       //a cluster without the user script should not be able to import the arrow library
-      withNewNotebook(ronCluster) { notebookPage =>
-        notebookPage.executeCell("""print 'Hello Notebook!'""") shouldBe Some("Hello Notebook!")
-        notebookPage.executeCell("""import arrow""").get should include("ImportError: No module named arrow")
-      }
-
+//      withNewNotebook(ronCluster) { notebookPage =>
+//        notebookPage.executeCell("""print 'Hello Notebook!'""") shouldBe Some("Hello Notebook!")
+//        notebookPage.executeCell("""import arrow""").get should include("ImportError: No module named arrow")
+//      }
       // create a new bucket, add the user script to the bucket, create a new cluster using the URI of the user script and create a notebook that will check if the user script ran
-      withNewGoogleBucket(billingProject) { bucketName =>
+      val gpAllocScriptProject = claimGPAllocProject(hermioneCreds)
+      val billingScriptProject = GoogleProject(gpAllocScriptProject.projectName)
+      withNewGoogleBucket(billingScriptProject) { bucketName =>
+        val ronPetServiceAccount = Sam.user.petServiceAccountEmail(billingProject.value)(ronAuthToken)
+        googleStorageDAO.setBucketAccessControl(bucketName, GcsEntity(ronPetServiceAccount, GcsEntityTypes.User), GcsRoles.Owner)
+
         val userScriptString = "#!/usr/bin/env bash\n\npip2 install arrow"
         val userScriptObjectName = GcsObjectName("user-script.sh")
         val userScriptUri = s"gs://${bucketName.value}/${userScriptObjectName.value}"
 
         withNewBucketObject(bucketName, userScriptObjectName, userScriptString, "text/plain") { objectName =>
+          googleStorageDAO.setObjectAccessControl(bucketName, objectName, GcsEntity(ronPetServiceAccount, GcsEntityTypes.User), GcsRoles.Owner)
           val clusterName = ClusterName("user-script-cluster" + makeRandomId())
-
           withNewCluster(billingProject, clusterName, ClusterRequest(Map(), None, Option(userScriptUri))) { cluster =>
+            Thread.sleep(10000)
```

Comment:
The test breaks without this, my guess is that this has something to do with the new chromedriver version